modules/mapping-utils/src/resolveSets.js

```diff
@@ -8,24 +8,40 @@ const retrieveSetIds = async ({
   type,
   query,
   path,
+  sort,
   BULK_SIZE = 1000,
 }) => {
   const search = async ({ searchAfter } = {}) => {
     const body = {
       ...(!isEmpty(query) && { query }),
       ...(searchAfter && { search_after: searchAfter }),
     };
+
     const response = await es.search({
       index,
       type,
-      sort: ['_id'],
+      sort: sort.map(
+        ({ field, order }) => `${field}${order ? `:${order}` : ''}`,
+      ),
       size: BULK_SIZE,
       body,
     });
     const ids = response.hits.hits.map(x =>
-      get(x, `_source.${path.split('__').join('.')}`),
+      get(x, `_source.${path.split('__').join('.')}`, x._id || ''),
+    );
+
+    const sortColValues = sort.map(({ field }) =>
+      response.hits.hits.map(x => x._source[field] || x[field]),
     );
```

Comment:
I _think_ this map + the reduce below is equivalent to:
```
  const searchAfter = sort
    .map(({ field }) => response.hits.hits.map(x => x._source[field] || x[field]))
    .map(x => x.slice(-1))
```
& just use `searchAfter` in the args below... if you feel like it lol

Reply:
can't do the ` .map(x => x.slice(-1))` part cuz it'll come back with a nested array [['stuff'],['moreStuff']], need a flat map but didn't want to import anything so just used reduce

Reply:
ohhh right you are, nvm ma bad